### PR TITLE
test: check-users: expect disabled 'Terminate session' button for logged in users

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by KKoukiou — COMMENTED** [review]
The commit message is wrong, should be 'enabled', please change on merge.

**Review by KKoukiou — COMMENTED** [review]
@marusak for the root user the button should be always disabled according to the code. https://github.com/cockpit-project/cockpit/blob/main/pkg/users/account-details.js#L272

File: test/verify/check-users [hunk 1]
```diff
@@ -84,10 +84,7 @@ class TestAccounts(MachineCase):
         b.wait_visible("#account-delete[disabled]")
         b.wait_visible("#account-real-name[disabled]")
         # when running against existing machines there might be an ssh root session
```
Comment (marusak) [inline test/verify/check-users:86]: This comment is now pointless, isn't it?

a41aca6fded2caaba672769cc5c38a6dbb8bfb88 introduced this change which was released in version 216. Indeed code in this version did not do any checks for `uid === 0`, that was only introduced in 5df053cad5eb37919bbdc5ecaaa02911e5b3a980 which was released in version 221. So this all indeed makes sense, thank you!

**Review by marusak — APPROVED** [review]
Thank you!

**Comment by KKoukiou** [comment]
The one test failing on centos stream on tmt fails also in other PRs https://github.com/cockpit-project/cockpit/pull/17959

**Pull request merged**